packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -341,3 +440,45 @@ const CommentBoard = ( {
 		</>
 	);
 };
+
+const ResolutionMessage = ( { entry } ) => {
+	// Fetch user information based on userId.
+	const user = useSelect(
+		( select ) => {
+			if ( ! entry.userId ) {
+				return null;
+			}
+			return select( coreStore ).getUser( entry.userId );
+		},
+		[ entry.userId ]
+	);
+
+	const getActionMessage = ( action ) => {
+		if ( 'resolve' === action ) {
+			return __( 'Marked as resolved' );
+		} else if ( 'reopen' === action ) {
+			return __( 'Re-opened' );
+		}
+		return '';
+	};
+
+	return (
+		<>
```

Comment:
There's no Fragment here. The `VStack` is the only element returned by this component.